README.md

```diff
@@ -7,18 +7,58 @@ The code here includes regression test data of computation units coming from ser
 
 **WARNING** This repo is under active development and relies on code and data that is not publicly available at this point.
 
-## Getting started
+## QuickStart
 
-Use the `tests` target of the Makefile to run the unit tests.
+1. Ensure you have docker installed and available for building and running and has access to the VCM cloud
+
+You should enable running docker as your user rather than as root with sudo. First, create a docker group if it doesn't exist:
+
+```shell
+$ sudo groupadd docker
+```
+
+Add your user to this group:
+
+```shell
+$ sudo usermod -aG docker $USER
+```
+
+Logout/restart to ensure these changes take effect.
+
+Finally, authorize docker to pull from GCP:
+
+```shell
+$ gcloud auth configure-docker
+```
```

Comment:
I suggest using links to upstream documentation to keep the document shorter, while also providing full instructions to people who need them.

```suggestion
Be sure to complete any required post-installation instructions (e.g. [for linux](https://docs.docker.com/engine/install/linux-postinstall/)). Also [authorize Docker to pull from gcr](https://cloud.google.com/container-registry/docs/advanced-authentication). Your user will need to have read access to the `us.gcr.io/vcm-ml` repository.
```